Day7n8n9n10/github-api-project-ts/src/containers/SearchGithubUsers/SearchGithubUsers.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import { SelectByCity, Card } from "../../components";
+import useFetchUserByCity from "../../hooks/useFetchUsersByCity";
+import { Loader } from "../../components";
+import data from "../../utils/state-city-data.json";
+import classes from "./searchGithubUsers.styles.module.css";
+import Header from "./SearchGithubUsersUI/Heading";
+
+const SearchGithubUsers = () => {
+  const [states, setStates] = useState<Array<string>>([]);
+  const [selectedState, setSelectedState] = useState<string>(
+    "Andaman and Nicobar Islands"
+  );
+  const [selectedCity, setSelectedCity] = useState<string>("Gurugram");
```

Comment:
better not to hardcode  the city. can pass as an empty string and handle the empty string if it is breaking the page somewhere.

Reply:
Yes, nice suggestion. Will fix it.